example/go.mod

```diff
@@ -0,0 +1,10 @@
+module basm
+
+go 1.22.6
+
+require (
+	github.com/blocky/basm-go-sdk v0.0.0-20250301001938-49297c34b318
```

Comment:
This will need to get updated. If you have a suggestion on a better way to deal with this, I'm all ears.

Reply:
I think you may be able to use the "replace" directive.  See https://go.dev/ref/mod#go-mod-file-replace

Reply:
updated to use replace, thank you for the suggestion. I also think a go workspace might be a solution but I can't tell yet if we should use one over the other.

Reply:
I ended up deleting the mod files here as we want the example to depend on the source code in development, not a pinned version